projects/js-packages/publicize-components/src/components/social-previews/modal.js

```diff
@@ -79,7 +79,8 @@ export default withSelect( select => {
 
 	const postData = {
 		post: getCurrentPost(),
-		title: getEditedPostAttribute( 'title' ),
+		title:
+			getEditedPostAttribute( 'meta' )?.jetpack_seo_html_title || getEditedPostAttribute( 'title' ),
```

Comment:
To match the style of the subsequent line, should the `getEditedPostAttribute()` call be on the subsequent line?

```suggestion
			getEditedPostAttribute( 'meta' )?.jetpack_seo_html_title ||
			getEditedPostAttribute( 'title' ),
```

Unsure if the next line is just because it had /quite/ so many options, or if this may as well split too.

Reply:
Yeah, Prettier wanted it on the one line 💅